python/test/unit/language/test_core_amd.py

```diff
@@ -1511,7 +1511,8 @@ def test_gemm(SIZE_M, SIZE_N, SIZE_K, NUM_WARPS, BLOCK_SIZE_M, BLOCK_SIZE_N, BLO
                         stride_cm=c.stride(0), stride_cn=c.stride(1),
                         M=a.shape[0], N=b.shape[1], K=a.shape[1],
                         BLOCK_SIZE_M=BLOCK_SIZE_M, BLOCK_SIZE_N=BLOCK_SIZE_N, BLOCK_SIZE_K=BLOCK_SIZE_K,
-                        num_warps=NUM_WARPS)
+                        num_warps=NUM_WARPS,
+                        num_stages=2)
```

Comment:
The change here is to explicitly tune on pipeline, so to be consistent with before, i.e., num_stages=2, right?

Reply:
yes, I use this test to check that slicing is handled properly in `shared layout` -> `mfma dot operand layout` conversion,
without pipelining sliced tensors are not generated in this test.

Reply:
I want to introduce some dedicated test for slice tensors.
But we have this test, so I think it is not urgent.